jekyll/_includes/snippets/find-organization-id.adoc

```diff
@@ -1,6 +1,6 @@
 [NOTE]
 ====
-The organization ID is required, and is provided with the `--owner-id` flag. To find your organization/user ID, select **Organization Settings** in the CircleCI web app side bar.
```

Comment:
I've just removed this part from the snippet to make the snippet more generic so it can be used in more places, before it was just used for config policies docs, which is why it specified this flag

Reply:
Ah yes I checked this and I think it all still makes sense